tests/fixtures/ssl/Makefile

```diff
@@ -0,0 +1,8 @@
+ssl_keys: clear_keys
+	@CNS="client client2" ./create_keys.sh
```

Comment:
Probably less error prone to just pass the CNs as arguments to create_keys.sh than relying on an env var